src/main/java/com/ef/cim/objectmodel/KeycloakUser.java

```diff
@@ -19,7 +20,7 @@
 @NoArgsConstructor
 @AllArgsConstructor
 @ToString
-public class KeycloakUser {
```

Comment:
What is this needed?

Reply:
I have used Conversation object as the data for CONVERSATION_HOLD timer, the Conversation object is referring to CCUser which then uses the KeycloakUser, this was giving Serialization error while writing the data to JMS.